src/subsys/ngx_subsys_lua_misc.c.tt2

```diff
@@ -216,8 +216,8 @@ ngx_[% subsys %]_lua_ngx_set(lua_State *L)
 }
 
 
-#ifndef NGX_LUA_NO_FFI_API
 [% IF http_subsys %]
+#ifndef NGX_LUA_NO_FFI_API
```

Comment:
Since this module defines `NGX_LUA_NO_FFI_API`, we need to introduce the new FFI function outside of this branch. However, we still want the ngx_http_lua version of this new FFI function to be injected within this branch.

This is the case thanks to this patch.